StyleCop.Analyzers/StyleCop.Analyzers/Helpers/DiagnosticOptionsHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace StyleCop.Analyzers.Helpers
+{
+    using System.Collections.Immutable;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.Diagnostics;
+
+    /// <summary>
+    /// Provides helper methods to work with diagnostics options.
+    /// </summary>
+    internal static class DiagnosticOptionsHelper
+    {
+        internal static bool IsAnalyzerEnabled(this SyntaxNodeAnalysisContext context, string analyzerId)
```

Comment:
:bulb: I would prefer to see this method be changed to `IsAnalyzerSuppressed`. The lack of a suppression does not mean the analyzer is enabled.